Packs/SumoLogic_Cloud_SIEM/Integrations/SumoLogicCloudSIEM/SumoLogicCloudSIEM.py

```diff
@@ -168,6 +171,18 @@ def get_update_result(resp_json):
     return {'Result': 'Success' if resp_json is True else 'Failed', 'Server Response': resp_json}
 
 
+def insight_timestamp_to_created_format(timestamp_int):
+    '''
+    Querying Insights using 'created' as opposed to 'timestamp' requires this conversion
+    '''
+    try:
+        created_time = datetime.utcfromtimestamp(timestamp_int)
+    except ValueError:
+        demisto.error('Error when converting timestamp to created format. Value: {}'.format(timestamp_int))
+        created_time = datetime.utcnow()
```

Comment:
not sure this is the right way to handle a failure as this might cause to missing events since we are looking at the now time instead of the last fetch time

Reply:
Agreed, this was mostly for testing actually - we should not get any unexpected format in this field. Updated.